WalletWasabi.Tests/RegressionTests/DosTests.cs

```diff
@@ -269,15 +269,15 @@ public async Task NotingTestsAsync()
 
 			Uri baseUri = new(RegTestFixture.BackendEndPoint);
 
-			List<(BitcoinWitPubKeyAddress changeOutputAddress, BlindedOutputWithNonceIndex blindedData, InputProofModel[] inputsProofs)> registerRequests = new();
-			AliceClient4 aliceClientBackup = null;
+			List<(BitcoinAddress changeOutputAddress, BlindedOutputWithNonceIndex blindedData, InputProofModel[] inputsProofs)> registerRequests = new();
+			AliceClient4? aliceClientBackup = null;
 			CoordinatorRound round = coordinator.GetCurrentInputRegisterableRoundOrDefault();
 			for (int i = 0; i < roundConfig.AnonymitySet; i++)
 			{
-				BitcoinWitPubKeyAddress activeOutputAddress = new Key().PubKey.GetSegwitAddress(network);
-				BitcoinWitPubKeyAddress changeOutputAddress = new Key().PubKey.GetSegwitAddress(network);
+				var activeOutputAddress = new Key().PubKey.GetAddress(ScriptPubKeyType.Segwit, network);
```

Comment:
It seems one can use

```suggestion
				var activeOutputAddress = new Key().GetAddress(ScriptPubKeyType.Segwit, network);
```

here (and other places as well) per https://github.com/MetacoSA/NBitcoin/blob/72f7ac92cd5b76aeaab8fd585a85d8623692b743/NBitcoin/Key.cs#L53. If applicable, it may be a good follow-up PR too.

Minor difference is that `network` is null checked.

Reply:
Done. Thx.